src/test/java/com/sublinks/sublinksapi/customemoji/services/CustomEmojiServiceTest.java

```diff
@@ -18,7 +18,7 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
Please review the style guide: https://sublinks.org/docs/style_guide.html

Wildcard imports are not allowed.

Reply:
Sorry about that miss, I've reviewed all the code for styleguide violations, I believe I've covered everything this time